src/operation/get-parameters-as-json-schema.js

```diff
@@ -605,7 +222,5 @@ module.exports = (path, operation, oas, globalDefaults = {}) => {
     });
 };
 
-module.exports.constructSchema = constructSchema;
```

Comment:
This export was only here to facilitate some easier test construction in `get-parameters-as-json-schema.test.js`, and wasn't and isn't used anywhere else or within our codebase. I'm not going to force a breaking change for this.